index.html

```diff
@@ -282,7 +283,7 @@ <h3 class="bannerbox__title">I need a ride</h3>
 
             <!-- http://40.117.45.210/driver/ -->
             <div id="pnlOfferRide">
-                <form name="offerRide" action="https://api.carpoolvote.com/v1.1/driver" method="post" class="form-horizontal" data-toggle="validator" role="form">
+                <form name="offerRide" action="http://localhost:8000/driver" method="post" class="form-horizontal" data-toggle="validator" role="form">
```

Comment:
@jkbits1 This form action is still set to localhost. I assume that's leftover test code?